src/vm/internal/compiler/lex.cc

```diff
@@ -841,10 +846,82 @@ static int skip_to(const char *token, const char *atoken) {
   }
 }
 
+void init_include_path() {
+  svalue_t *ret;
```

Comment:
For new code ,please definitely code in C++ 11 style,  we are in 2015 now, writing code in pre-C89 format is only harming readability , not improving it. 

Move declaration to their use, and use STL / unique_pointer whenever possible. and most importantly, don't use goto!